langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -201,6 +264,16 @@ public isolated function setLength((any|error)[] arr, int length) returns () = @
 # Returns `()` if not found.
 # Equality is tested using `==`.
 #
+# ```
+# string[] greetings = ["Hello", "Hola", "Bonjour", "Hola", "Ciao"];
+#
+# // First index of "guten tag" if it exists in the list.
+# greetings.indexOf("guten tag") => ()
```

Comment:
Output is currently the `toString` representation except where the value is nil since for nil toString is the empty string and we can't represent it here. Alternatively, we can probably do something like
```suggestion
# greetings.indexOf("guten tag") is () => true
```

Reply:
+1

Reply:
+1

Reply:
Updated both occurrences in [037bdb0](https://github.com/ballerina-platform/ballerina-lang/pull/38989/commits/037bdb0467633b46a5fb089b26ac4dd84cdf4bf6).